web/core/store/issue/cycle/issue.store.ts

```diff
@@ -185,7 +185,7 @@ export class CycleIssues extends BaseIssuesStore implements ICycleIssues {
       // get params from pagination options
       const params = this.issueFilterStore?.getFilterParams(options, cycleId, undefined, undefined, undefined);
       // call the fetch issues API with the params
-      const response = await this.cycleService.getCycleIssues(workspaceSlug, projectId, cycleId, params, {
+      const response = await this.issueService.getIssues(workspaceSlug, projectId, params, {
```

Comment:
You should add the cycle id to the params.

Reply:
That is being handled in the `const params = this.issueFilterStore?.getFilterParams` at line 186